snuba/cli/entities.py

```diff
@@ -60,3 +62,12 @@ def describe(entity_name: str) -> None:
         entity.describe().accept(CLIDescriber())
     except InvalidEntityError:
         click.echo(f"Entity {entity_name} does not exists or it is not registered.")
+
+
+@entities.command()
+@click.argument("entity_path", type=str)
+def load(entity_path: str) -> None:
```

Comment:
What is this used for? Would it be used in the testing workflow when a developer adds a new entity?

Reply:
exactly, just a quick way to validate your YAML if you haven't written tests yet